packages/devtools_app/lib/src/app.dart

```diff
@@ -263,16 +271,21 @@ class DevToolsAppState extends State<DevToolsApp> {
 
   @override
   Widget build(BuildContext context) {
-    return ValueListenableBuilder(
-      valueListenable: widget.preferences.darkModeTheme,
+    return ValueListenableBuilder<bool>(
```

Comment:
instead of nesting ValueListenableBuilders here and above, let's just add a listener in initState that listens to the value listenable and stores the value in DevToolsAppState class. You can do that for both the vmDeveloperModeEnabled and darkModeTheme value listenables

Reply:
Done.